### DISCUSSION TITLE
How to use a GitHub repository as a webpage? （GitHub Pages）

### DISCUSSION THREAD

**Discussion by djl777** [docsifyjs/docsify#2280] [Q&A]
## Problem

I want to use a specific **GitHub Repository** to deploy a static webpage (as I've already used xxx.github.io). I use [docsify](https://docsify.js.org/#/) to deploy it. This is my related **repo**(https://github.com/djl777/note), and this is my **website**(https://djl777.github.io/note/#/) generated.

When I upload my folder, I find some trouble: my avg and font are not changed, the relative code is in https://github.com/djl777/note/blob/master/index.html: 
```html
 <link rel="icon" href="/static/image/avator.jpg">

<!-- font -->
  <link rel="stylesheet" href="/static/css/font.css">
```
So I change them to:
```html
 <link rel="icon" href="/note/static/image/avator.jpg">

<!-- font -->
  <link rel="stylesheet" href="/note/static/css/font.css">
```
and it does works, but does not work in my **local port** becauses of the ‘note’, so this is not a good idea.

I want to know why the root directory is xxx.github.io instead of xxx.github.io/note, and how to change it.

## supplement(another problem)
Today I find another problem in https://github.com/djl777/note/blob/master/README.md. 
```markdown
![test1](/static/image/avator.jpg)

![test2](/note/static/image/avator.jpg)
```
You could see my [homepage](https://djl777.github.io/note/#/) that the **test1(/static/image/avator.jpg) is ok but not test2(/note/static/image/avator.jpg)**, this disproves my previous conjecture about absolute paths, and I'm very confused as to why `index.html` behaves differently than `README.md`. Is this not a github page issue but something to do with docify?

Besides, my setting is:
![image](https://github.com/community/community/assets/73270782/be4cfd40-4171-4628-80d4-025ed85dcee6)

**Comment by sy-records** [comment]
In HTML files, the / symbol represents the "root directory".

In docsify, see #1891.

I chose to use the cdn address directly in the quote.

https://github.com/lufei/notes/blob/master/docs/images/hyperf/hyperf-1.png

```

![](https://cdn.jsdelivr.net/gh/lufei/notes@master/docs/images/hyperf/hyperf-1.png)
```

**Reply by djl777** [reply]
您好，忽略图片的路径问题（可能与docify有关），我觉得应该是我使用github仓库作为网页的路径有问题？
参照我上面的描述，当我将仓库设置为github page时，根目录是 xxx.github.io/ 而不是某个仓库  xxx.github.io/note/，而且我之前使用另一款笔记页面 mkdocs 进行部署也遇到了一样的问题，才让我怀疑是 github page 的问题，是不是我 github page 的配置有问题呢？
